modules/widgets/simple-payments/customizer.js

```diff
@@ -0,0 +1,294 @@
+
+			var imageContainer = widgetForm.find( '.jetpack-simple-payments-image' );
+
+			var mediaFrame = new wp.media.view.MediaFrame.Select( {
+				title: 'Choose Product Image',
+				multiple: false,
+				library: { type: 'image' },
+				button: { text: 'Choose Image' },
+			} );
+
+			mediaFrame.on( 'select', function() {
+				var selection = mediaFrame.state().get( 'selection' ).first().toJSON();
+				//hide placeholder
+				widgetForm.find( '.jetpack-simple-payments-image-fieldset .placeholder' ).hide();
+
+				//load image from media library
+				imageContainer.find( 'img' )
+					.attr( 'src', selection.url )
+					.show();
+
+				//show image and remove button
+				widgetForm.find( '.jetpack-simple-payments-image' ).show();
+
+				//set hidden field for the selective refresh
+				widgetForm.find( '.jetpack-simple-payments-form-image-id' ).val( selection.id ).change();
+			} );
+
+			mediaFrame.open();
+		};
+	}
+
+	function removeImage( widgetForm ) {
+		return function( event ) {
+			event.preventDefault();
+
+			//show placeholder
+			widgetForm.find( '.jetpack-simple-payments-image-fieldset .placeholder' ).show();
+
+			//hide image and remove button
+			widgetForm.find( '.jetpack-simple-payments-image' ).hide();
+
+			//set hidden field for the selective refresh
+			widgetForm.find( '.jetpack-simple-payments-form-image-id' ).val( '' ).change();
+		};
+	}
+
+	function isFormValid( widgetForm ) {
+		var errors = false;
+		var postTitle = widgetForm.find( '.jetpack-simple-payments-form-product-title' ).val();
+		if ( ! postTitle ) {
+			widgetForm.find( '.jetpack-simple-payments-form-product-title' ).addClass( 'invalid' );
+			errors = true;
+		}
+
+		var productPrice = widgetForm.find( '.jetpack-simple-payments-form-product-price' ).val();
+		if ( ! productPrice ) {
+			widgetForm.find( '.jetpack-simple-payments-form-product-price' ).addClass( 'invalid' );
+			errors = true;
+		}
+
+		var productEmail = widgetForm.find( '.jetpack-simple-payments-form-product-email' ).val();
+		var isProductEmailValid = /^((([a-z]|\d|[!#\$%&'\*\+\-\/=\?\^_`{\|}~]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+(\.([a-z]|\d|[!#\$%&'\*\+\-\/=\?\^_`{\|}~]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+)*)|((\x22)((((\x20|\x09)*(\x0d\x0a))?(\x20|\x09)+)?(([\x01-\x08\x0b\x0c\x0e-\x1f\x7f]|\x21|[\x23-\x5b]|[\x5d-\x7e]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(\\([\x01-\x09\x0b\x0c\x0d-\x7f]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]))))*(((\x20|\x09)*(\x0d\x0a))?(\x20|\x09)+)?(\x22)))@((([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))\.)+(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))\.?$/i.test( productEmail );
```

Comment:
This is likely a copy pasta from elsewhere, but where did we get this one from? As a warning getting email validation right in a regex is rather difficult (the official spec is quite complex) so oftentimes folks can only be certain if we email something and it works. So expect a few false positive and negatives with valid/invalid emails, even if this looks quite long.